services/database-analyzer.js

```diff
@@ -37,28 +48,49 @@ function DatabaseAnalyzer(databaseConnection, config) {
           if (foreignKey
             && foreignKey.foreign_table_name
             && foreignKey.column_name && !columnInfo.primaryKey) {
+            let alias = _.camelCase(foreignKey.column_name);
+            if (alias.endsWith('Id') && alias.length > 2) {
+              alias = alias.substring(0, alias.length - 2);
+            } else if (alias.endsWith('Uuid') && alias.length > 4) {
+              alias = alias.substring(0, alias.length - 4);
+            }
             const reference = {
               ref: foreignKey.foreign_table_name,
               foreignKey: foreignKey.column_name,
-              as: `_${foreignKey.column_name}`,
+              as: alias,
             };
 
             if (foreignKey.foreign_column_name !== 'id') {
               reference.targetKey = foreignKey.foreign_column_name;
             }
 
+            log(`  Detected ${chalk.bold('relationship')} ${chalk.magenta(reference.as)}`);
+            log(`    with ${chalk.bold('reference to table')} ${chalk.cyan(reference.ref)}`);
+            log(`    with ${chalk.bold('foreign key')} ${chalk.cyan(reference.foreignKey)}`);
+            if (reference.targetKey) {
+              log(`    with ${chalk.bold('target key')} ${chalk.cyan(reference.targetKey)}`);
+            }
             references.push(reference);
           } else if (type) {
             // NOTICE: If the column is of integer type, named "id" and primary, Sequelize will
             //         handle it automatically without necessary declaration.
             if (!(columnName === 'id' && type === 'INTEGER' && columnInfo.primaryKey)) {
               const field = {
+                cameCaseName: _.camelCase(columnName),
                 name: columnName,
                 type,
                 primaryKey: columnInfo.primaryKey,
                 defaultValue: columnInfo.defaultValue,
               };
 
+              const typeOfField = field.primaryKey ? 'primary key' : 'field';
+              log(`  Detected ${chalk.bold(typeOfField)} ${chalk.magenta(field.cameCaseName)}`);
+              log(`    with ${chalk.bold('column name')} ${chalk.cyan(field.name)}`);
+              log(`    with ${chalk.bold('type')} ${chalk.cyan(field.type)}`);
+              if (field.defaultValue !== undefined) {
+                log(`    with ${chalk.bold('defaultValue')} ${chalk.cyan(field.defaultValue)}`);
+              }
```

Comment:
one line per field would be lighter and easier to read
Something like:
```
Added model "User" (table: 'users')
  Added field "id" (column: "id", type: INTEGER)`
  Added field "fullName" (column: "full_name", type: STRING, default value: "vince")`
```

And for a better experience with mongoose, we should replace "column" by "field"

Reply:
Why did you should to use 3 or 4 different colors in the logs?

Reply:
Just to detect easily the important information